### PR TITLE
use string_view in create_torrent

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

File: src/create_torrent.cpp [hunk 1]
```diff
@@ -75,7 +76,7 @@ namespace libtorrent
 			return 0;
 #else
 			struct stat s;
-			if (lstat(convert_to_native(p).c_str(), &s) < 0) return 0;
+			if (lstat(convert_to_native(std::string(p)).c_str(), &s) < 0) return 0;
```
Comment (arvidn) [inline src/create_torrent.cpp]: is there a ``to_string()`` for ``string_view``?

Comment (aldenml) [inline src/create_torrent.cpp]: yes :) and you already use it https://github.com/arvidn/libtorrent/blame/master/src/kademlia/node.cpp#L779

Comment (arvidn) [inline src/create_torrent.cpp]: ah, it's not a free function..

File: src/create_torrent.cpp [hunk 2]
```diff
@@ -665,28 +666,28 @@ namespace libtorrent
 		return dict;
 	}
 
-	void create_torrent::add_tracker(std::string const& url, int tier)
+	void create_torrent::add_tracker(string_view url, int const tier)
 	{
-		m_urls.push_back(announce_entry(url, tier));
+		m_urls.push_back(announce_entry(std::string(url), tier));
```
Comment (aldenml) [inline src/create_torrent.cpp]: here?

**Pull request merged**